airbyte-integrations/connectors/source-hubspot/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1.49",
+    "airbyte-cdk~=0.1.55",
```

Comment:
Just pointing out that I noticed only hubspot has a patch version in the requirements. Others either don't have a version at all, or are set to `~=1`